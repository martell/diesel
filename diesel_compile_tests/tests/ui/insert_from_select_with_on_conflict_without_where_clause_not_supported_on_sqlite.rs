extern crate diesel;

use diesel::*;

table! {
    users {
        id -> Integer,
    }
}

fn main() {
    let connection = SqliteConnection::establish("").unwrap();

    users::table.select(users::id)
        .insert_into(users::table)
        .into_columns(users::id)
        .on_conflict(users::id)
        .do_nothing()
        .execute(&connection)
        .unwrap();

}
