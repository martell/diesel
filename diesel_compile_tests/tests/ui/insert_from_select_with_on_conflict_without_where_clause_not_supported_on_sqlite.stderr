error[E0271]: type mismatch resolving `<diesel::SqliteConnection as diesel::Connection>::Backend == diesel::pg::Pg`
  --> $DIR/insert_from_select_with_on_conflict_without_where_clause_not_supported_on_sqlite.rs:19:10
   |
19 |         .execute(&connection)
   |          ^^^^^^^ expected struct `diesel::sqlite::Sqlite`, found struct `diesel::pg::Pg`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0271`.
